src/blocks/pacman.rs

```diff
@@ -254,7 +256,7 @@ fn get_pacman_available_updates() -> Result<String> {
 
     // Update database
     run_command(&format!(
-        "fakeroot -- pacman -Sy --dbpath \"{}\" --logfile /dev/null &> /dev/null",
+        "pacman -Sy --dbpath \"{}\"",
```

Comment:
I'm not sure why we had `--logfile /dev/null` in the first place, but do you really need to remove it to fix your issue? If not then might be better to leave it for now, and perhaps remove it later once it's proven that it is really not needed.

Reply:
all output is discarded anyways and this just keeps the command simpler

Reply:
> do you really need to remove it to fix your issue

technically I don't but since I was there I just did since it had no effect.

Reply:
OK so it looks like we're just doing exactly what the `checkupdates` script does, so I would prefer to keep it for now even though it doesn't appear to be needed. There might be some weird issue that it solves.